header.php

```diff
@@ -31,7 +31,7 @@
 
 		<a class="skip-link sr-only sr-only-focusable" href="#content"><?php esc_html_e( 'Skip to content', 'understrap' ); ?></a>
 
-		<nav class="navbar navbar-expand-md navbar-dark bg-primary">
+		<nav class="navbar navbar-expand-md navbar-dark bg-primary <?php echo (!get_theme_mod('understrap_header_position')) ?: 'fixed-top' ?>">
```

Comment:
Missing the ending semi-colon and also some PHPCS formatting adjustments:

`echo  ( ! get_theme_mod( 'understrap_header_position' ) ) ?: 'fixed-top';`